### ISSUE TITLE
Ability to display longitude, latitude, altitude

### ISSUE THREAD

**Issue by alexanderbock** [OpenSpace/OpenSpace#382]
It is possible to `goToGeo`, but no ability to display the geo location.

**Comment by kbladin** [comment]
I have thought about a lua function `getGeoPosition` for the globe browsing module. Is it possible for it to return a `vec3` or perhaps a lua table defining a `Geodetic3`? The function `lua_pushnumber` can be used to push a single number to the lua state but how can I do it with many numbers so that the function returns more than one number?

**Comment by alexanderbock** [comment]
```lua
   lua_newtable(L);

    lua_pushnumber(L, lat);
    lua_rawseti(L, -2, 1); 
    lua_pushnumber(L, long);
    lua_rawseti(L, -2, 2); 
    lua_pushnumber(L, alt);
    lua_rawseti(L, -2, 3); 
    return 1;
```

`lua_rawseti`  the second argument is the relative position of the table on the stack (-1: the `lua_pushnumber` number, -2, the table from `lua_newtable`) and the third argument is the index where you want the number to be

**Comment by alexanderbock** [comment]
Alternative:

```lua
lua_pushnumber(L, lat);
lua_pushnumber(L, long);
lua_pushnumber(L, alt);

return 3;
```

and then in Lua:
```lua
lat, long, alt = openspace.globebrowsing.getGeoPosition()
```

**Comment by kbladin** [comment]
Ah! Easy :)

**Comment by kbladin** [comment]
getGeoPosition lua function available in #380
